guides/getting-started.md

```diff
@@ -136,15 +136,25 @@ Note that `md-icon` supports any font or svg icons; using Material Icons is one
 ## Appendix: Configuring SystemJS
 
 If your project is using SystemJS for module loading, you will need to add `@angular/material` and
-`@angular/cdk` to the SystemJS configuration:
+`@angular/cdk` to the SystemJS configuration.
+
+For the `@angular/cdk`, the different utilities are split into separate entry points. 
```

Comment:
Can also say "must be" there. I chose this phrasing, though, to make it seem like less "this is something extra you must do" and more "this is the way it is".